Dockerfile

```diff
@@ -1,7 +1,18 @@
-FROM bentoml/model-server:0.11.0-py37
+FROM bentoml/model-server:0.11.0-py38
 MAINTAINER ersilia
 
 RUN pip install rdkit
+RUN pip install git+https://github.com/bp-kelley/descriptastorus
+RUN pip install tqdm>=4.62.2
```

Comment:
I don't think we should need tqdm because it is just a visual progress bar, and while running the model as a server, we do not need any visual components.